gulpfile.js

```diff
@@ -39,7 +40,10 @@ var REACT_OPTS = {es6: true},
 // Runs JSX, ES6 transforms, browserify, adds sourcemaps, and notifies the
 // livereload server that JS has changed.
 gulp.task('js', function() {
-  var bundler = watchify(browserify(BROWSERIFY_OPTS));
+  livereactload.monitor(PATHS.dest + 'bundled.js')
+
+  var bundler = watchify(browserify(_.extend({transform: [livereactload]}, BROWSERIFY_OPTS)));
+  rebundle();
```

Comment:
It's callable at this point (you can call functions earlier in the file from where they're defined; seems quite common in JS (in fact, this is a copy-pasta error). Killed the extra rebundle() call though :)